CHANGELOG.md

```diff
@@ -0,0 +1,98 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
+
+## [Unreleased]
+
+NOTICE: The ABI has changed, please make sure to update your nwnx.nss and recompile all your scripts. 
+
+### Added
+- New plugins: Area, CombatModes, Effect, Encounter, Feedback, Rename, Visibility
+- Added support to skip building certain plugins by setting an environment variable, for example: `NWNX_SKIP_PLUGINS="JVM;Lua;Mono;Ruby;SpellChecker"`
+- Added support to build out of tree plugins with the following environment variable: `NWNX_ADDITIONAL_PLUGINS={pathtoplugins}`
```

Comment:
Perhaps better: `NWNX_ADDITIONAL_PLUGINS=/path/to/Plugin1;/path/to/Plugin2` so the `{}` don't confuse people, and also so it shows how to add more than one.